app/uploaders/picture_uploader.rb

```diff
@@ -0,0 +1,22 @@
+class PictureUploader < CarrierWave::Uploader::Base
+  include CarrierWave::MiniMagick
+  storage :fog
+
+  def store_dir
+    "images/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+  process resize_to_fill: [360, 360]
+
+  version :thumb do
+    process resize_to_fill: [100, 100]
+  end
+
+  def extension_white_list
+    %w(jpg jpeg gif png)
+  end
+
+  def default_url(*args)
+    ActionController::Base.helpers.asset_path("fallback/" + [version_name, "default.jpg"].compact.join('_'))
```

Comment:
Line is too long. [108/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.